src/riak_core_ssl_util.erl

```diff
@@ -198,15 +209,16 @@ verify_ssl(Cert, valid_peer, {App, MyCommonName}) ->
                 "certificate's common name: ~p", [CommonName]),
             {fail, duplicate_common_name};
         _ ->
-            case validate_common_name(CommonName,
-                    app_helper:get_env(App, peer_common_name_acl, "*")) of
+            ACL = app_helper:get_env(App, peer_common_name_acl, "*"),
+            case validate_common_name(CommonName, ACL) of
                 {true, Filter} ->
-                    lager:info("SSL connection from ~s granted by ACL ~s",
+                    lager:info("SSL connection from ~s granted by ACL \"~s\"",
                         [CommonName, Filter]),
```

Comment:
At a glance, looks like this should be `ACL`, not `Filter` (but haven't looked at `validate_common_name` yet).